luamml-amsmath.lua

```diff
@@ -44,23 +44,71 @@ lua.get_functions_table()[funcid] = function()
   set_row_attribute('columnalign', token.scan_argument())
 end
 
+-- This function is used to add a intent :continued-row to
+-- rows of a split environment.
+-- we assume that the table is a mtable with mrow with mtd. 
+-- we check row 2..n. If the first cell has only one element and
+-- for this element 'tex:ignore' has been set, we assume a continued row and
+-- set the intent on the mrow.
+local function add_intent_continued_row (table)
+  for index,rowtable in ipairs(table) do
+   if table[index][1] and table[index][1][1] then -- just for safety ...
+     if index > 1 and #table[index][1]==1 and table[index][1][1]['tex:ignore'] then
+      table[index]['intent']=':continued-row'
+     end
+   end
+  end
+end
+
+-- This function add an intent =":pause-medium" on every second mtd in a table
+-- currently it is also on the first (after the label) but this could be changed
+-- used in __luamml_amsmath_finalize_table:n for 
+-- 'align' or 'alignat' or 'flalign' or  'xalignat' or 'xxalignat'
```

Comment:
argubably it shouldn't pause for the `...at` versions as they don't add space but often users add a `\quad` or similar space there anyway (using alignat for user-set space rather than no-space) so when I tried this on some examples it seemed OK, possibly though we will need to come back to that at some point.